src/containers/subjects-launch/SubjectWTSStarAlignment/index.tsx

```diff
@@ -0,0 +1,275 @@
+import React, { useState } from 'react';
+import { useMutation } from 'react-query';
+import { isEqual } from 'lodash';
+import { DataTable } from 'primereact/datatable';
+import { Column } from 'primereact/column';
+import { Button } from 'primereact/button';
+import { Dialog } from 'primereact/dialog';
+import { RadioButton } from 'primereact/radiobutton';
+
+import CircularLoaderWithText from '../../../components/CircularLoaderWithText';
+import StyledJsonPretty from '../../../components/StyledJsonPretty';
+
+import { invokeWTSSAWorkflow, StarAlignmentPayload } from './aws';
+import { usePortalSubjectDataAPI } from '../../../api/subject';
+import { FastqRow, usePortalFastqAPI } from '../../../api/fastq';
+import { usePortalMetadataAPI } from '../../../api/metadata';
+import JSONToTable from '../../../components/JSONToTable';
+
+const metadataHeaderToDisplay: string[] = [
+  'subject_id',
+  'sample_id',
+  'library_id',
+  'external_subject_id',
+  'external_sample_id',
+  'phenotype',
+  'type',
+  'assay',
+  'source',
+  'project_name',
+  'project_owner',
+];
+
+type Props = { subjectId: string };
+export default function SubjectLaunchWTSStarAlignment({ subjectId }: Props) {
+  const [isConfirmDialogOpen, setIsConfirmDialogOpen] = useState<boolean>(false);
+
+  const [input, setInput] = useState<StarAlignmentPayload | null>(null);
+
+  // Find out the associated LibraryId
+  const {
+    isLoading: isLoadingSubjectData,
+    isError: isErrorSubjectData,
+    data: subjectData,
+  } = usePortalSubjectDataAPI(subjectId);
+  const listOfLibrary = subjectData?.lims.map((o) => o.library_id);
```

Comment:
Will; could you also filter `Type==WTS`, please. Star Alignment is only valid for WTS library type, yes.